extensions/vale-styleguide/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "suse-vale-styleguide",
+  "displayName": "SUSE Vale rules",
+  "description": "Vale rules for SUSE styleguide",
+  "version": "0.1.0",
```

Comment:
Do you raise the version afterwards? I think this change would qualify for a higher version (maybe 0.2.0?)

Reply:
it's the initial version of the extension, why not 0.1.0?

Reply:
Oh, I see! In that case, 0.1.0 is perfectly fine.